Sources/PointFree/Episode/Episode.swift

```diff
@@ -98,18 +112,35 @@ private let episodeTocView = View<[Episode.TranscriptBlock]> { blocks in
   )
 }
 
-private let tocChapterView = View<(content: String, timestamp: Double)> { content, timestamp in
+private func timestampLinkAttributes(_ timestamp: Int) -> [Attribute<Element.A>] {
+  return [
+    href(""),
+
+    onclick(javascript: """
+    var video = document.getElementsByTagName("video")[0];
+    video.currentTime = event.target.dataset.t;
+    video.play();
+    event.preventDefault();
+    """),
```

Comment:
lol this is fun. I literally inline this javascript into each chapter link, which makes the video jump to a particular timestamp.